src/invoices/InvoiceDetails/InvoiceDetailsContainer.js

```diff
@@ -62,12 +68,38 @@ function InvoiceDetailsContainer({
             },
           });
           const approvalsConfigPromise = mutator.invoiceActionsApprovals.GET();
+          const batchVoucherExportPromise = mutator.batchVoucherExport.GET({
+            params: {
+              limit: `${LIMIT_MAX}`,
+              query: `batchVouchers.batchedVouchers=folioInvoiceNo:${invoiceResponse.folioInvoiceNo}`,
+            },
+          });
+          const exportConfigsPromise = mutator.exportConfigs.GET({
+            params: {
+              limit: `${LIMIT_MAX}`,
+              query: `batchGroupId==${invoiceResponse.batchGroupId}`,
+            },
+          });
 
-          return Promise.all([vendorPromise, invoiceLinesPromise, approvalsConfigPromise]);
+          return Promise.all([
+            vendorPromise,
+            invoiceLinesPromise,
+            approvalsConfigPromise,
+            batchVoucherExportPromise,
+            exportConfigsPromise,
+          ]);
         })
-        .then(([vendorResp, invoiceLinesResp, approvalsConfigResp]) => {
+        .then(([
+          vendorResp,
+          invoiceLinesResp,
+          approvalsConfigResp,
+          batchVoucherExportResp,
+          exportConfigsResp,
+        ]) => {
           setVendor(vendorResp);
           setInvoiceLines(invoiceLinesResp);
+          setBatchVoucherExport(batchVoucherExportResp[0]);
+          setExportFormat(exportConfigsResp[0]?.format);
```

Comment:
IMHO, it's kind a wrong logic. if config has no format or it's absent we shouldn't display table at all

Reply:
`format` is required for exportConfig